ServerCore/Pages/Puzzles/Play.cshtml

```diff
@@ -178,6 +184,12 @@ else
                             }
                         </td>
                     }
+                    else
+                    {
+                        <td>
+                            <component type="typeof(ServerCore.Pages.Components.PresenceComponent)" render-mode="Server" param-PuzzleUserId="@Model.LoggedInUser.ID" param-TeamId="@Model.Team.ID" param-PuzzleId="@item.ID" param-IsReadOnly=true />
```

Comment:
The full list of names might get crowded on the puzzle list if several players join a puzzle. Consider shortening it. Maybe something like for 3+ players, show a count?